python/cudf/cudf/_libxx/strings/char_types.pxd

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+from libcpp.memory cimport unique_ptr
+from cudf._libxx.cpp.column.column_view cimport column_view
+from cudf._libxx.cpp.column.column cimport column
+
+cdef extern from "cudf/strings/char_types/char_types.hpp" \
+        namespace "cudf::strings" nogil:
+
+    ctypedef enum string_character_types:
```

Comment:
Lets change the underlying C++ to strongly type this enum class with `int32_t` underneath, and then define an additional `ctypedef` in the Cython here above the extern block for use with future Python Enums

Reply:
@kkraus14, Made the changes.